dev/devicelab/lib/tasks/platform_channels_benchmarks.dart

```diff
@@ -39,15 +39,13 @@ TaskFunction runTask(adb.DeviceOperatingSystem operatingSystem) {
         '-d',
         device.deviceId,
       ];
-      return utils.startFlutter(
+      final Process flutterProcess = await utils.startFlutter(
         'run',
         options: options,
       );
+      final Map<String, double> results = await microbenchmarks.readJsonResults(flutterProcess);
```

Comment:
I assume microbenchmarks.readJsonResults can work from a different directory but I dont actually know.

Reply:
as far as i can tell it works entirely from the provided process' stdout and has no dependency on the current working directory.

Reply:
That works for me. Thanks for taking this bug :)